credit-card-recommendation/src/model_training.py

```diff
@@ -0,0 +1,95 @@
+import time
+import mlflow
+import numpy as np
+import pandas as pd
+from sklearn.preprocessing import LabelEncoder, StandardScaler
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import roc_auc_score
+from sklearn.linear_model import LogisticRegression
+
```

Comment:
should we add some on requrements.txt from above import declaration?

Reply:
Done.